inst/www/htmlwidgets.js

```diff
@@ -576,29 +578,30 @@
       forEach(matches, function(el) {
         var sizeObj = initSizing(el, binding);
 
+        var getSize = function(el) {
+          if (sizeObj) {
+            return {w: sizeObj.getWidth(), h: sizeObj.getHeight()}
+          } else {
+            var rect = el.getBoundingClientRect();
+            return {w: rect.width, h: rect.height}
+          }
+        };
+
         if (hasClass(el, "html-widget-static-bound"))
           return;
         el.className = el.className + " html-widget-static-bound";
 
+        var lastSize = getSize(el);
+
         var initResult;
         if (binding.initialize) {
-          initResult = binding.initialize(el,
-            sizeObj ? sizeObj.getWidth() : el.offsetWidth,
-            sizeObj ? sizeObj.getHeight() : el.offsetHeight
-          );
+          initResult = binding.initialize(el, lastSize.w, lastSize.h);
           elementData(el, "init_result", initResult);
         }
 
         if (binding.resize) {
```

Comment:
Need to set `lastSize` here, as the size could've changed as a result of initialization